packages/editor/src/plugins/image-gallery/utils/create-empty-image-plugin.tsx

```diff
@@ -0,0 +1,26 @@
+import { EditorPluginType } from '@editor/types/editor-plugin-type'
+import { v4 as uuidv4 } from 'uuid'
+
+export function createEmptyImagePlugin(url: string, caption: string) {
```

Comment:
I'm confused about this one. Some questions:

- Why do you init the caption even though it's optional?
- Did you try not adding the uuids manually? I think they should be added if they are missing.

Reply:
It's called `createEmptyImagePlugin`, why does it accept url and caption?

Reply:
Did you try `state.images.insert(0, { plugin: EditorPluginType.Image })` above instead of this function?

Reply:
Removed.